src/hybrid_reader.c

```diff
@@ -228,6 +259,7 @@ static size_t HR_LastDocId(void *ctx) {
 static void HR_Rewind(void *ctx) {
   HybridIterator *hr = ctx;
   hr->resultsPrepared = false;
+  hr->numIterations = 0;
```

Comment:
I think it is better to have here `numIterations++` so developers can see it in FT.PROFILE.

Reply:
Agreed. Will change